examples/meta-counter/src/contract.rs

```diff
@@ -40,8 +41,10 @@ impl Contract for MetaCounterContract {
         // Send a no-op message to ourselves. This is only for testing contracts that send messages
         // on initialization. Since the value is 0 it does not change the counter value.
         let this_chain = self.runtime.chain_id();
-        self.runtime
-            .emit(StreamName(b"announcements".to_vec()), b"instantiated");
+        self.runtime.emit(
+            StreamName(b"announcements".to_vec()),
+            &b"instantiated".to_vec(),
```

Comment:
This will eventually by serialize `Self::EventValue`?

Reply:
Ah, it already is - it's just this particular example.

Reply:
This _is_ `Self::EventValue`. I made it `Vec<u8>` for the meta counter. Looking at this, maybe `String` would make more sense?